backend/config/ssl/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
This self-signed cert is only used in development so it is ok to check in to git.